src/components/Card/CardList.tsx

```diff
@@ -55,7 +68,7 @@ export function CardList({ cardItemData }: CardListProps) {
         <CardItem
           key={card.link}
           data={card}
-          handleSwipe={(dir) => swiped(dir, card.title)}
+          handleSwipe={(dir) => swiped(dir, card.title, card.link)}
```

Comment:
Really sorry I have adjusted this to be `const swiped = (direction: string, card: CardItemData) => {}` now, if you look at the other PR